### PR TITLE
Change updated registrations notification to put

### PR DESCRIPTION
**Pull request by feclare**

### CODE REVIEW

File: client/registration.go [hunk 1]
```diff
@@ -253,6 +253,15 @@ func delRegByName(w http.ResponseWriter, r *http.Request) {
 func notifyUpdatedRegistrations() {
 	go func() {
 		// TODO make configurable distro host/port
-		http.Get("http://127.0.0.1:48070/api/v1/notify/registrations")
+		client := &http.Client{}
+		req, err := http.NewRequest(http.MethodPut, "http://127.0.0.1:48070/api/v1/notify/registrations", nil)
+		if err != nil {
+			logger.Info("Error creating http request")
```
Comment (drasko) [inline client/registration.go]: Why not logging in the `Error` mode here, but only `Info`?

File: client/registration.go [hunk 2]
```diff
@@ -253,6 +253,15 @@ func delRegByName(w http.ResponseWriter, r *http.Request) {
...
+			return
+		}
+		_, err = client.Do(req)
+		if err != nil {
+			logger.Info("Error notifying updated registrations to distro")
```
Comment (drasko) [inline client/registration.go]: Please log as error, not as info.

**Comment by feclare** [comment]
Done

File: client/registration.go [hunk 3]
```diff
@@ -253,6 +253,15 @@ func delRegByName(w http.ResponseWriter, r *http.Request) {
 func notifyUpdatedRegistrations() {
 	go func() {
 		// TODO make configurable distro host/port
-		http.Get("http://127.0.0.1:48070/api/v1/notify/registrations")
+		client := &http.Client{}
+		req, err := http.NewRequest(http.MethodPut, "http://127.0.0.1:48070/api/v1/notify/registrations", nil)
```
Comment (drasko) [inline client/registration.go]: Please do not hard-code the addresses.  `host` and `port` should be const types, defined earlier.

**Comment by feclare** [comment]
Done, they are cont now. But in the future they need to be configurable.

**Comment by drasko** [comment]
Merged, thanks!

**Pull request merged**